### DISCUSSION TITLE
Logic App Standard -  Working locally with VS Code and renaming Workflow files

### DISCUSSION THREAD

**Discussion by ventil8** [Azure/logicapps#749] [Help Needed] [answered]
We are moving to Logic Apps Standard and as part of this want to move the Workflows themselves to code/repos.

We have terraform code to deploy the Logic App and then pipelines to deploy the Workflows into those Logic app. So one Logic App will have many workflows.

using VS code I can view this setup in azure then Create a new project from it. I am asked to name the stateless Logic app which then creates an empty workflow.json. I can than copy the code into this JSON file and hey presto I have 1 of my required workflows. I can right click and use designer mode.

However, and this is my issue. As soon as I rename this from workflow.json to the name i want, i no longer have the ability to use the designer? and if i cant rename it I cant add other workflows to this logic app.

This doesn't seem right to me...or am i doing something wrong?

**Comment by mark-abrams** [comment, marked as answer]
The name of the workflow is indicated by the name of the folder containing the `workflow.json`. You don't need to rename the workflow file, and you should never do this.

**Comment by ventil8** [comment]
Thanks Mark, makes sense. Will adjust my thinking on this.
